src/routes/index.js

```diff
@@ -1,9 +1,16 @@
-import { Router } from 'express';
-import api from './api';
+import express from 'express';
+import userRoute from './api/users';
+import authRoutes from './api/auth';
+import resetPasswordRoute from './api/resetPassword';
+import accommodationRoute from './api/accommodation';
+import roomRoute from './api/room';
 
+const router = express.Router();
 
-const router = new Router();
-
-router.use('/api/v1', api);
```

Comment:
@sa-ma welldone,  is there a reason you removed this block of code from here??

Reply:
Yeah, Regle doesn't track where it was kept before. I gave an explanation in the slack channel